src/app/lib/config/services/afrique.js

```diff
@@ -148,32 +149,37 @@ const service = {
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'Read about our approach to external linking.',
+        text: 'Découvrez notre approche en matière de liens externes.',
       },
       links: [
         {
           href: 'https://www.bbc.com/news/help-41670342',
-          text: 'Why you can trust the BBC',
+          text: 'Pourquoi vous pouvez faire confiance à BBC News',
```

Comment:
Looks to have been updated to https://www.bbc.com/afrique/institutionelles-49283281

Reply:
Sorry, this should be for the link of cource